WordPress/Classes/ViewRelated/Stats/Insights/Latest Post Summary/LatestPostSummaryCell.swift

```diff
@@ -146,13 +147,13 @@ private extension LatestPostSummaryCell {
         }
 
         let postAge = lastPostInsight?.publishedDate.relativeStringInPast() ?? ""
-        let postTitle = lastPostInsight?.title.nonEmptyString() ?? NSLocalizedString("(No Title)", comment: "Empty Post Title")
+        postTitle = lastPostInsight?.title.nonEmptyString() ?? NSLocalizedString("(No Title)", comment: "Empty Post Title")
```

Comment:
It's unfortunate that the optionality propagates to lines 152 & 156.

If we change line 33 to:
```
private(set) var postTitle = NSLocalizedString("(No Title)", comment: "Empty Post Title")
```

then this segment might be simplified to something like:
```
if let title = lastPostInsight?.title, !title.isEmpty {
    postTitle = title
}
```

And we can remove the `!` on the aforementioned lines.

Thoughts?